src/ParseFile.js

```diff
@@ -25,7 +25,7 @@ export type FileSource = {
 };
 
 var dataUriRegexp =
-  /^data:([a-zA-Z]*\/[a-zA-Z+.-]*);(charset=[a-zA-Z0-9\-\/\s]*,)?base64,/;
+  /^data:([\w\/\+]+);(charset=[a-zA-Z0-9\-\/\s]*,)?base64,/;
```

Comment:
@AppOrchestra Nice catch regarding the inability to match digits. Although the way you've set up this regex would also match things with an arbitrary # of slashes, like `too//many//slashes/here`. In addition `\w` (if I'm correct regarding the regex flavor) will probably match `_` as well, which is not specifically desired.

To avoid the included `_` character with `\w`, and to factor in digits, you could probably do something like:
```
[-a-zA-Z0-9]+\/[-a-zA-Z0-9+.]+
```
This basically uses the existing regex, but makes a couple changes. It adds a `-` to the first part of the regex, allowing for valid mime types with a `-` at the start, like `x-conference/x-cooltalk` (which is a thing, although rare). It also changes the quantifier `*` for a `+`, specifying that we expect at least one or more characters before and after the slash, something like `/mimetype` or `audio/` shouldn't be considered valid; the existing regex would allow this.

You could also just use `\d` instead of `0-9`, totally up to you, just wanted to give you a heads up on that.